velox/exec/Spill.h

```diff
@@ -147,15 +156,19 @@ class FileSpillMergeStream : public SpillMergeStream {
   }
 
   int32_t numSortKeys() const override {
+    VELOX_CHECK(!closed_);
```

Comment:
No need to check in this method

Reply:
We get the sortKeys and compareFlags from spillFile_, after close, the spillFile is reset, so I check it here.